### ISSUE TITLE
the meaning of the dataset

### ISSUE THREAD

**Issue by LS11111** [kooksung/pathological_gait_datasets#3]
Hello,thanks for your dataset.
but I am filled with some confusion about the meaning of the dataset.
such as human1_antalgic1-20 
From my guess,
human 1 perform antalgic 20 times
but I don't know that  how to sample in one antalgic gait to generate human1_antalgic1_SkeletonData1-6.
The timing of human1_antalgic1_SkeletonData1 to human1_antalgic1_SkeletonData2 does not seem to be continuous.
Looking forward to your reply!

**Comment by LS11111** [comment]
Do human1_antalgic1_SkeletonData1-6 show the data collected from 6 different Kinect respectively?
What do I need to deal with the data?mixture?

**Comment by kooksung** [comment]
Hello,

Thank you for the interest to our dataset.

We developed 6 Kinects data collecting system to cover 8-m walkway.

So, after walking 10-m walkway, 6 data instances were collected. (human1_antalgic1_SkeletonData1-6-->8-m walking once).

Do human1_antalgic1_SkeletonData1-6 show the data collected from 6 different Kinect respectively? --> Yes.
What do I need to deal with the data? mixture? --> you don’t have to mix the data. you can think 6 data instances as individual data (unless you evaluate the data with intra-subject method).

In summary, 10 humans , 6 gait patterns , 20 walkings (8-m) = 120 instances.

You can read the paper: https://ieeexplore.ieee.org/abstract/document/9152991/

If you have additional questions, feel free to contact me.

Thank you.

Sincerely,
Kooksung Jun
